### PR TITLE
threejs: cameras in ThreeJS

### PR DESCRIPTION
**Pull request by nguyend-nam**
#### What's this PR do?

- [x] Add basic information about cameras in ThreeJS

### CODE REVIEW

File: Engineering/ThreeJS/Cameras in ThreeJS.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
+---
+tags: engineering/threejs, threejs, camera
+author: Nguyen Dinh Nam
+date: 2022-09-15
+---
+
+ThreeJS is a JavaScript 3D library that allows developers to develop and describe data in 3 dimensions, and then convert them into 2 dimensions and display them on HTML Canvas. **Camera** is one of the core elements of a ThreeJS project, beside Scene, Renderer and 3D objects.
```
[the remark this answers is not in the document]
Comment (ngolapnguyen) [reply Engineering/ThreeJS/Cameras in ThreeJS.md:7]: It'd be great if we can also have the articles for them in `brain`, covering all the basic of ThreeJS. But at the moment I think using external docs is good enough.

File: Engineering/ThreeJS/Cameras in ThreeJS.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
+---
+tags: engineering/threejs, threejs, camera
```
Comment (ngolapnguyen) [inline Engineering/ThreeJS/Cameras in ThreeJS.md:2]: I think this should be `engineering/frontend/threejs`.

File: Engineering/ThreeJS/Cameras in ThreeJS.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+Place them into the Oxyz coordinate...
+
+![](https://i.imgur.com/uIocTAX.png)
+
+### Perspective camera
+
+![](https://i.imgur.com/OIpfGw7.png)
+
+Some crucial attributes:
+- **fov** (field of view): Camera frustum vertical field of view, i.e. the angle of the view in the yOz plane, in degree. The lower the number, the narrower the view.
```
Comment (ngolapnguyen) [inline Engineering/ThreeJS/Cameras in ThreeJS.md:28]: Would be great if we have an illustration for fov in 3D space. Understanding fov in 3D context could be difficult.

**Review by ngolapnguyen — CHANGES_REQUESTED** [review]
Might also consider adding in some existing ThreeJS samples (or your own CodePen) showcasing `PerspectiveCamera` and `OrthographicCamera`. I think that would help making the points of this article much clearer.

**Review by zlatanpham — APPROVED** [review]
LGTM. @monotykamary might we test the image plugin here?

**Comment by monotykamary** [comment]
ORPHAN COMMIT FOR REFLOG: [0c9b6aa20a4fc5c56de015b372ec9e8b732580a6](https://github.com/dwarvesf/brain/commits/0c9b6aa20a4fc5c56de015b372ec9e8b732580a6)

**Comment by monotykamary** [comment]
> LGTM. @monotykamary might we test the image plugin here?

Looks all good 👍 

```diff
From d3d4f5d7097a2c89a92b0c7124d0a108d58bbc3b Mon Sep 17 00:00:00 2001
From: Tom Nguyen <tom81094@gmail.com>
Date: Wed, 21 Sep 2022 11:51:39 +0700
Subject: [PATCH] chore: generate local images

---
 Engineering/ThreeJS/Cameras in ThreeJS.md |  10 +++++-----
 _assets/OIpfGw7.png                       | Bin 0 -> 467251 bytes
 _assets/Va74b4E.png                       | Bin 0 -> 159737 bytes
 _assets/ZKOiIim.png                       | Bin 0 -> 505216 bytes
 _assets/cX1U9zI.png                       | Bin 0 -> 507573 bytes
 _assets/uIocTAX.png                       | Bin 0 -> 377219 bytes
 6 files changed, 5 insertions(+), 5 deletions(-)
 create mode 100644 _assets/OIpfGw7.png
 create mode 100644 _assets/Va74b4E.png
 create mode 100644 _assets/ZKOiIim.png
 create mode 100644 _assets/cX1U9zI.png
 create mode 100644 _assets/uIocTAX.png

diff --git a/Engineering/ThreeJS/Cameras in ThreeJS.md b/Engineering/ThreeJS/Cameras in ThreeJS.md
index bc61010..2936f70 100644
--- a/Engineering/ThreeJS/Cameras in ThreeJS.md	
+++ b/Engineering/ThreeJS/Cameras in ThreeJS.md	
@@ -14,19 +14,19 @@ You cannot use the `Camera` class directly to initialize a new camera in ThreeJS
 
 The image below shows the object viewed in **perspective camera** (upper) and in **orthographic camera** (lower).
 
-![](https://i.imgur.com/Va74b4E.png)
+![](_assets/Va74b4E.png)
 
 Place them into the **Oxyz** coordinate...
 
-![](https://i.imgur.com/uIocTAX.png)
+![](_assets/uIocTAX.png)
 
 ### Perspective camera
 
-![](https://i.imgur.com/OIpfGw7.png)
+![](_assets/OIpfGw7.png)
 
 Some crucial attributes:
 - **fov** (field of view): Camera frustum **vertical** field of view, i.e. the angle of the view in the yOz plane, in degree. The lower the number, the narrower the view.
-![](https://i.imgur.com/ZKOiIim.png)
+![](_assets/ZKOiIim.png)
 - **near** and **far**: Clipping planes at 2 points on the z axis. Note that objects with z position outside the `(near, far)` interval will **not** be displayed.
 
 ```javascript
@@ -42,7 +42,7 @@ Perspective camera visualization from [r105.threejsfundamentals.org](https://r10
 
 ### Orthographic camera
 
-![](https://i.imgur.com/cX1U9zI.png)
+![](_assets/cX1U9zI.png)
 
 Some crucial attributes:
 - **left**, **right**, **top** and **bottom**: Horizontal and vertical position of 4 segments of the view.
```

**Pull request merged**